src/styles/blocks/card.scss

```diff
@@ -0,0 +1,69 @@
+.product {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  background: $btn_color;
+  color: $main_color;
+
+  @include elementSpace;
```

Comment:
Don't add margins to block. Use BEM mix. Refer to the videos or checklist for more details.